### ISSUE TITLE
Dead links in readme.md

### ISSUE THREAD

**Issue by FelDev** [fooloomanzoo/datetime-picker#66]
[https://fooloomanzoo.github.io/datetime-picker/components/datetime-picker/
](https://fooloomanzoo.github.io/datetime-picker/components/datetime-picker/
)and
[https://fooloomanzoo.github.io/datetime-picker/components/datetime-picker/#/elements/datetime-picker/demos/demo/datetime-elements.html](https://fooloomanzoo.github.io/datetime-picker/components/datetime-picker/#/elements/datetime-picker/demos/demo/datetime-elements.html
)
Both yield 404

Basically, [https://fooloomanzoo.github.io](https://fooloomanzoo.github.io) is gone and now this repo lacks a demo.
